api/settings/dev.py

```diff
@@ -1,5 +1,15 @@
+import logging
+
 from .prod import *
 
 DEBUG = True
+ALLOWED_HOSTS = ['*']
+
+# optional setting to configure logs to print on stdout
+for logger_name, logger_dict in LOGGING['loggers'].items():
+    logger_dict['handlers'] = ['console']
+logging.config.dictConfig(LOGGING)
 
-ALLOWED_HOSTS = ['*']
\ No newline at end of file
+# hack to disable autoreload logger
+l = logging.getLogger('django.utils.autoreload')
```

Comment:
Just use: `logging.getLogger('django.utils.autoreload').disabled = True`  
unless we are using `l` for `None` check